tests/stubs/README.md

```diff
@@ -3,4 +3,20 @@
 The files here are used for the unit tests for BRO code that's applicable. 
 Each file correlates to a unit test method and contains the mock data that will be used for filtering.
 Because each test method is targeting a specific filter scenario, we need the results to be uniform and assertable.
-This is the main reason we opt to create a new set of YAML for each test method and scenario.
\ No newline at end of file
+This is the main reason we opt to create a new set of YAML for each test method and scenario.
+
+## Unit Test Function & Stub File Example
+
```

Comment:
I think you can simplify this explanation (everything under "#Unit Test Function & Stub File Example), like so:

There's a 1:1 correspondence between each test function in `collector_test.go` and its input YAML in `test/stubs`. For example, the test `TestFilterByName_ExcludeRegexWithWildcardInclude` will get its YAML input from `test/stubs/TestFilterByName_ExcludeRegexWithWildcardInclude.yaml`, and that YAML is then unmarshalled into a slice of `unstructured.Unstructured` objects, which the filters expect to see. Each test constructs its own `v1.ResourceSelector` and verifies that the objects filtered by that selector are as expected.

Reply:
Updated the explanation to be a lot more in line with this one. LMK if you'd like it to be exactly word for word, or if you'd like other edits.